src/db_adapters/account_changes.rs

```diff
@@ -1,11 +1,25 @@
 use actix_diesel::dsl::AsyncRunQueryDsl;
 use diesel::PgConnection;
+use futures::future::try_join_all;
 
 use crate::models;
 use crate::schema;
 
-/// Saves state change related to account to database
 pub(crate) async fn store_account_changes(
+    pool: &actix_diesel::Database<PgConnection>,
+    shards: &[near_indexer::IndexerShard],
+    block_hash: &near_indexer::near_primitives::hash::CryptoHash,
+    block_timestamp: u64,
+) -> anyhow::Result<()> {
+    let futures = shards
+        .iter()
+        .map(|shard| store_state_changes(pool, &shard.state_changes, block_hash, block_timestamp));
+
+    try_join_all(futures).await.map(|_| ())
+}
+
+/// Saves state change related to account to database
+async fn store_state_changes(
```

Comment:
nit: in other files it's called `..._for_chunk`, let's call it `store_account_changes_for_chunk`?

Reply:
But it is for shard this time, `state_changes` are not in `chunk`, but I got you

Reply:
@khorolets you renamed not the function @telezhnaya was talking about. The top-level function should be called `store_account_changes` while the helper function that stores the account changes for a single shard: `store_account_changes_for_shard`